roles/translator/src/upstream_sv2/upstream.rs

```diff
@@ -370,6 +433,22 @@ impl Upstream {
         });
         Ok(())
     }
+    fn get_job_id(
+        self_: &Arc<Mutex<Self>>,
+    ) -> Result<Result<u32, crate::error::Error<'static>>, crate::error::Error<'static>> {
+        self_
+            .safe_lock(|s| {
+                if s.is_work_selection_enabled() {
+                    // TODO MVP3 this will be set by SetCustomMiningJobSuccess
```

Comment:
A github issue would be better.

Reply:
This is already defined by MVP3 not sure that we need an issue for that